ogre2/src/Ogre2SelectionBuffer.cc

```diff
@@ -181,10 +184,27 @@ void Ogre2SelectionBuffer::Update()
 /////////////////////////////////////////////////
 void Ogre2SelectionBuffer::DeleteRTTBuffer()
 {
-  if (this->dataPtr->selectionMaterial)
+  // \todo Delete material?
+  // This causes an ogre assertion error about unlinking renderables
+  // when the RTT buffer is created again
+  // if (this->dataPtr->selectionMaterial)
+  // {
+  //   Ogre::MaterialManager::getSingleton().remove(
+  //       this->dataPtr->selectionMaterial->getName());
+  //   this->dataPtr->selectionMaterial.setNull();
+  // }
```

Comment:
Are you planning to remove these comments before merging?

Reply:
should be fixed. ab5b397